src/applications/managing-app-lifecycle.md

```diff
@@ -98,6 +98,22 @@ From deis-controller.local:peachy-waxworks
  * [new branch]      master     -> deis/master
 ```
 
+
+## Application Maintenance
+
+Maintenance mode for applications is useful to perform certain migrations or upgrades during which we don't want to serve client requests. Deis Workflow supports maintenance mode for app during which the access to the app is blocked. Blocking access to the application means all the requests to the app are served with error code of `503` and static maintenance page by the router but the app will still be running and one-off commands can still be run on them.
```

Comment:
Need to add articles (a, an, the) before three nouns:
> maintenance mode for *an* app
> with *an* error code of `503` and *a* static maintenance page

Also I would remove the last two words "on them".